classes/Task.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+// Класс, представляющий сущность "Задание"
+class Task
+{
+    private const STATUS_NEW = 'statusNew';
+    private const STATUS_CANCELED = 'statusCanceled';
+    private const STATUS_IN_WORK = 'statusInWork';
+    private const STATUS_PERFORMED = 'statusPerformed';
+    private const STATUS_FAILED = 'statusFailed';
+    private const ACTION_CANCEL = 'actionCancel';
+    private const ACTION_RESPOND = 'actionRespond';
+    private const ACTION_EXECUTE = 'actionExecute';
+    private const ACTION_REFUSE = 'actionRefuse';
+
+    private string $currentStatus = self::STATUS_NEW;
+
+    public function __construct(
+        private readonly int $customerId,
+        private $executorId = null
+    ) {
+    }
+
+    /**
+     * Объект класса создаёт заказчик, исполнитель, скорее всего, подключится позже
+     * Функция установки id исполнителя задания
+     * @param int $executorId - id исполнителя задания
+     * 
+     * @return int - id исполнителя задания
+     */
+    public function setExecutorId(int $executorId): self
+    {
+        $this->executorId = $executorId;
+
+        return $this;
+    }
+
+    /**
+     * Функция возвращения текущего статуса задания
+     * 
+     * @return string - текущий статус задания
+     */
+    public function getCurrentStatus(): string
+    {
+        return $this->current_status;
```

Comment:
Тут будет ошибка - имя свойства `currentStatus`
```suggestion
        return $this->currentStatus;
```